.github/workflows/playwright.yml

```diff
@@ -23,6 +23,8 @@ jobs:
       - uses: actions/setup-node@v4
         with:
           node-version: ${{ matrix.node-version }}
+      - name: Install Yarn
```

Comment:
So did we do this in CS3D as well? I thought we just used bun instead there. We should be consistent in any case.